sources/documents/2_RTB/Verbali/Esterni/VerbaleEsterno_2024-05-06/capitolo-riunione.tex

```diff
@@ -0,0 +1,59 @@
+\section{Riunione}
+\subsection{Ordine del giorno}
+\begin{itemize}
+	\item Discutere delle tecnologie per lo sviluppo della web app;
+	\item Esporre studio preliminare delle tecnologie lato back-end;
+	\item Valutare l'ampliamento dei casi d'uso;
+\end{itemize}
+
+\subsection{Argomenti e temi dell'incontro}
+
+\subsubsection{Tecnologie per lo sviluppo della web app}
+
+\textbf{Domanda:} Tra le diverse tecnologie solitamente indicate per lo sviluppo di un'applicazione web (quali Node.js, Flask etc.), abbiamo individuato il \glossario{framework} \glossario{Streamlit}. 
+Questo permette uno sviluppo \glossario{front-end} più celere rispetto alle alternative, seppur conceda meno spazio alla personalizzazione. 
+Secondo lei, potrebbe essere una scelta opportuna?
+
+\textbf{Risposta:} La \glossario{Proponente} non è familiare col framework Streamlit, ma ne approverebbe l'utilizzo a seguito di una presentazione informativa da parte del gruppo in cui si spiegano approfonditamente, ovvero tramite paragoni con gli altri framework, le motivazioni di tale scelta.
```

Comment:
La \glossario{Proponente} non è familiare col framework Streamlit, ma ne approverebbe l'utilizzo a seguito di una presentazione informativa da parte del gruppo in cui si spiegano le motivazioni di tale scelta.